utils/sn_flux_est.py

```diff
@@ -824,18 +967,27 @@ def est_pulsar_sn(pulsar, obsid,\
     logger.addHandler(ch)
     logger.propagate = False
 
-    if not args.obsid or not args.pulsar:
-        logger.error("Obsid and Pulsar name must be supplied. Exiting...")
-        sys.exit(1)
-
-    query = psrqpy.QueryATNF(psrs=[args.pulsar], loadfromdb=ATNF_LOC).pandas
-    #Decide what to use as ra and dec
-    if args.pointing is None:
-        raj = None
-        decj = None
+    if args.mode == "SNFE":
+        if not args.obsid or not args.pulsar:
+            logger.error("Obsid and Pulsar name must be supplied. Exiting...")
+            sys.exit(1)
+        pulsar = args.pulsar[0]
+        query = psrqpy.QueryATNF(psrs=[pulsar], loadfromdb=ATNF_LOC).pandas
+        #Decide what to use as ra and dec
+        if args.pointing is None:
+            raj = None
+            decj = None
+        else:
+            raj = args.pointing.split("_")[0]
+            decj = args.pointing.split("_")[1]
+
+        SN, SN_err = est_pulsar_sn(pulsar, args.obsid,\
+                beg=args.beg, end=args.end, p_ra=raj, p_dec=decj, plot_flux=args.plot_est, query=query, min_z_power=args.min_z_power)
+    elif args.mode == "ATNF":
+        if args.pulsar==None:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is None:'](https://app.codacy.com/manual/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4921142)